src/main.js

```diff
@@ -221,3 +217,17 @@ function addGrid() {
     // saveMyPosters.classList.add('mini-poster');
     openSavedPosters();
   }
+
+  function remove(node) {
```

Comment:
I think in our refactoring we should change the name of "node" since that has another meaning it might be confusing.